cpython/search_pypi_top.py

```diff
@@ -202,33 +235,21 @@ def _main():
         global IGNORE_CYTHON
         IGNORE_CYTHON = False
 
+    logging.basicConfig(format="# %(message)s")
```

Comment:
This change doesn't seem to be taken in account. I see:

```
...
grep PYPI-2022-01-26-TOP-5000/pep8-naming-0.12.1.tar.gz (23%, 1099/4762)
grep PYPI-2022-01-26-TOP-5000/SoundFile-0.10.3.post1.tar.gz (23%, 1100/4762)
...
```

The format is ignored.

Maybe initialize the logger variable after calling basicConfig()?

Reply:
Ah right. We can fix this by setting a formatter to a handler, and adding that. I've pushed a commit.